### DISCUSSION TITLE
[resolved] Zwiększenie zakresu offset na "Get a list of billing entries"

### DISCUSSION THREAD

**Discussion by Waclaw-W** [allegro/allegro-api#4907] [Suggestions - resolved]
Przy pobieraniu listy opłat nie przyjmuje offsetu więcej niz 1000

![Screenshot_51](https://user-images.githubusercontent.com/81759361/132640007-b666798b-1535-4766-bdc0-5819ba74dd5a.jpg)

Przydało by sie zwiększenie tego zakresu tak jak jest na innych listach np. GET seller offers ma offset 99999999

![Screenshot_50](https://user-images.githubusercontent.com/81759361/132639757-5169f3eb-0ca8-42ca-9d0c-639435b148dc.jpg)

Przy dużej ilości opłat aby zrobić listę miesięczną muszę teraz sumować/odptywać po dniach, a to raczej średnie rozwiązanie.

Czy komuś jeszcze przyda się takie usprawnienie ?

**Comment by jakubmaguza** [comment]
To jedna z bardziej plusowanych propozycji.
1000 to zdecydowanie za mało. Do jednego zamówienia pojawia się po kilkanaście pozycji.

**Comment by AureliuszBrussy** [comment]
Zwiększyliśmy offset. Więcej informacji znajdziecie [w newsie](https://github.com/allegro/allegro-api/issues/5392).
